app.js

```diff
@@ -16,4 +16,182 @@ Closer.addEventListener('click', () => {
+    ],
+    liveVersion:
+    'https://amklass11.github.io/Mobile-portfolio/',
+    source: 'https://github.com/amklass11/Mobile-portfolio',
+  },
+];
+function createCard(el) {
+  return `<div class="card white-bg">
+          <div class="card-img">
+            <img src="${el.image}" alt="${el.name}" />
+          </div>
+          <div class="card-det">
+            <h2>Tonic</h2>
+
+            <div class="card-list">
+              <h3>Canopy</h3>
+              <ul>
+                <li>lorem</li>
+                <li>lorem</li>
+              </ul>
+            </div>
+
+            <p class="desc">
+            ${el.description}
+            </p>
+
+            <div class="tags">
+              <span>${el.technologies[0]}</span> <span>${el.technologies[1]}</span> <span>${el.technologies[3]}</span>
+            </div>
+
+            <div class="button">
+              <button class="modal-button" type="button">See Project</button>
+            </div>
+          </div>
+        </div>`;
+}
+
+const myProjects = dataProjects.map((el, index) => {
+  const div = document.createElement('div');
+  div.className = 'work';
+  if (index % 2 === 1) {
+    div.classList.add('reverse');
+  }
+  div.innerHTML = createCard(el);
+  return div;
+});
+
+const worksSection = document.querySelector('.work');
+
+for (let i = 0; i < myProjects.length; i += 1) {
+  worksSection.appendChild(myProjects[i]);
+}
+
+function createModal(el) {
+  return `
+     <section id="work">
+<div class="modal-card white-bg">
+<div class="modal-card-det modal-head">
+<h2>Tonic <button class="lass" type="button">X</button></h2>
+<div class="modal-card-list">
+<h3>Canopy</h3>
+ <ul>
+                  <li>lorem</li>
+<li>lorem</li>
+ </ul>
+</div>
+          <div class="modal-card-img">
+            <img src="${el.image}" alt="${el.name}" />
+          </div>
+            <p class="desc-modal">
+              ${el.description}
+            </p>
+
+            <div class="tags">
+              <span>${el.technologies[1]}</span> <span>${el.technologies[2]}</span> <span>${el.technologies[3]}</span>
+            </div>
+
+            <div class="button">
+              <button type="button">See Project</button>
+            </div>
+          </div>
+        </div></section>
+      
+  `;
+}
+
+const modalBtn = document.querySelectorAll('.modal-button');
+const modalArea = document.querySelector('.modal');
+
+function openModal() {
+  modalArea.classList.toggle('display-modal');
+}
+
+function closeModal() {
+  modalArea.classList.toggle('display-modal');
+}
+
+modalBtn.forEach((el, index) => {
+  el.addEventListener('click', () => {
+    openModal();
+    const modalSection = document.createElement('div');
+    modalSection.className = 'ar';
+    modalSection.innerHTML = createModal(dataProjects[index]);
+    modalArea.appendChild(modalSection);
+    const closeModalBtn = document.querySelector('.lass');
+    closeModalBtn.addEventListener('click', closeModal);
+  });
 });
```

Comment:
- [ ] It would be nice if you write the logic for the modal to appear once on every click. Currently, upon the first clicking on any of the popup Modals, the one that is clicked will appear, and on the next click on any one, I noticed two Modals appear. Kindly fix that.

**Your design**
![image](https://user-images.githubusercontent.com/50941074/162577529-27661667-4906-4736-905f-19f17c2e5a25.png)